README.md

```diff
@@ -59,3 +59,23 @@ $ kubectl apply -f deploy/operatorsource.crd.yaml
 $ kubectl apply -f deploy/rbac.yaml
 $ kubectl apply -f deploy/csv/marketplace.csv.yaml
 ```
+
+## Using Operator Artifacts in an OperatorSource
+
+Follow the steps [here](./docs/how-to-upload-artifact.md) to upload an operator artifact to `quay.io`.
+
+Once your operator artifact is pushed to `quay.io` you can use an `OperatorSource` to add your operator offering to Marketplace. An example `OperatorSource` is provided [here](deploy/examples/operatorsource.cr.yaml).
+
+An `OperatorSource` must specify the `registryNamespace` the operator artifact was pushed to, and set the `name` and `namespace` for creating the `OperatorSource` on your cluster.
+
+Add your `OperatorSource` to your cluster:
+
+```bash
+$ oc create -f your-operator-source.yaml
+```
+
+Once created, the Marketplace operator will use the `OperatorSource` to download your operator artifact from the app registry and display your operator offering in the Marketplace UI.
+
+### Updating the OperatorSource
+
+Updating an `OperatorSource` is not supported. If you would like to update the `OperatorSource` CR, you must delete the previous `OperatorSource` resource, delete the marketplace pod and finally delete the package-server pod in the `openshift-operator-lifecycle-manager` namespace (this is to ensure that all cache and memory has been cleared - otherwise you may see stale operator data in the UI).
```

Comment:
We now support update of `OperatorSource` spec. The PR is being merged - https://github.com/operator-framework/operator-marketplace/pull/42

Reply:
Sounds good; I'll remove this section.